base/linalg/symmetric.jl

```diff
@@ -317,7 +317,7 @@ det(A::RealHermSymComplexHerm) = real(det(bkfact(A)))
 det(A::Symmetric{<:Real}) = det(bkfact(A))
 det(A::Symmetric) = det(bkfact(A))
 
-\(A::HermOrSym{<:Any,<:StridedMatrix}, B::StridedVecOrMat) = \(bkfact(A.data, Symbol(A.uplo), issymmetric(A)), B)
+\(A::HermOrSym{<:Any,<:StridedMatrix}, B::StridedVecOrMat) = \(bkfact(A), B)
```

Comment:
Didn't mean to include that here but that is essentially just replicating the method of `bkfact` on line 313 just above https://github.com/JuliaLang/julia/pull/22540/files#diff-a213a275abc68ea5a35f8c8332272829R313 so thought this would be cleaner.